lib/prompts/checkbox.js

```diff
@@ -86,7 +90,8 @@ Prompt.prototype.render = function (error) {
   var bottomContent = '';
 
   if (this.firstRender) {
-    message += '(Press <space> to select)';
+    message += '(Press <space> to select, <a> to select all,\n<n> to select none,';
```

Comment:
Okay so, let's simplify this case:

- space: select one
- a: select all/none
- i: inverse

Let's drop random.